apis/workloads/v1alpha1/replicatedstatemachine_types.go

```diff
@@ -30,18 +30,38 @@ import (
 
 // ReplicatedStateMachineSpec defines the desired state of ReplicatedStateMachine
 type ReplicatedStateMachineSpec struct {
-	// Replicas defines number of Pods
+	// replicas is the desired number of replicas of the given Template.
+	// These are replicas in the sense that they are instantiations of the
+	// same Template, but individual replicas also have a consistent identity.
+	// If unspecified, defaults to 1.
 	// +kubebuilder:default=1
 	// +kubebuilder:validation:Minimum=0
 	// +optional
-	Replicas int32 `json:"replicas,omitempty"`
+	Replicas *int32 `json:"replicas,omitempty"`
+
+	// selector is a label query over pods that should match the replica count.
+	// It must match the pod template's labels.
+	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
+	Selector *metav1.LabelSelector `json:"selector"`
```

Comment:
There are manage changes in the API.
Run `make manifests` and `make api-doc` to update clientset and API doc.

Reply:
done: [09375c2](https://github.com/apecloud/kubeblocks/pull/4086/commits/09375c28277be20a9780721a4de775ed38a96f96)